src/main/java/edu/kis/powp/jobs2d/drivers/adapter/Job2dDrawerAdapter.java

```diff
@@ -4,15 +4,17 @@
 import edu.kis.legacy.drawer.shape.ILine;
 import edu.kis.legacy.drawer.shape.LineFactory;
 import edu.kis.powp.jobs2d.Job2dDriver;
+import edu.kis.powp.jobs2d.features.DrawerFeature;
 
 /**
  * driver adapter to drawer with several bugs.
  */
-public class MyAdapter extends DrawPanelController implements Job2dDriver {
+public class Job2dDrawerAdapter implements Job2dDriver {
```

Comment:
The name is better but it does not indicate the purpose of the adapter or what was actually adapted.